tests/containers/podman_netavark.pm

```diff
@@ -64,7 +64,7 @@ sub _cleanup {
     validate_script_output('podman network ls', sub { /podman\s+bridge/ });
 }
 
-sub switch_to_netavark {
+sub chk_install_netavark_pkg {
```

Comment:
Why?

Reply:
The _original_ master version of switch routine had 2 phases:
n.1 install netavark and aadvark checking installation type transactional or not, 
n.2 restart system to make netavark new default.

But that  switch routine was called only when `is_cni` therefore not performed in a case, new added slem 5.5 skipped the is_cni test, but needing transactional aadvark, so that it failed in https://openqa.suse.de/tests/13535263#step/podman_netavark/120.

The `...vark` packages in this module should be _always_ present-or-installed-if-missing; then in next phase activated if is_cni was true, therfere the `....vark` install phase should be always executed(without package pre-selection, being automatically skipped if already present) in a **transactional check test**, to define the _install type_.

This last is performed in the new routine `chk_install_netavark_pkg` (that can be eventually renamed, if needed) splitted by old `switch_to_..`.  
The new `switch_to_..` routine contains now the only system restart phase.